addons/account/i18n/account.pot

```diff
@@ -492,11 +491,6 @@ msgid ""
 "#704A66;'>%(date)s</b>"
 msgstr ""
 
-#. module: account
-#: model_terms:ir.ui.view,arch_db:account.view_account_payment_register_form
-msgid "<b>Early Payment Discount of %(payment_reference) has been applied.</b>"
```

Comment:
This is weird. Double checking if this string should be correctly exported somehow

Reply:
Ah false alarm, it did export it as 2 separate terms as expected. Sorry I am blind :see_no_evil:

Reply:
I think it is exported properly as 2 strings:
`Early Payment Discount of`
`has been applied.`